### PR TITLE
SRCH-2331 the basic_affiliate fixture uses the default search engine

### PR DESCRIPTION
**Pull request by MothOnMars**
This PR updates the `basic_affiliate` fixture to use the default search engine (currently BingV7) instead of the deprecated Google search engine. It also includes updates to the VCR cassettes and some Rubocop fixes. The remaining Rubocop failures are preexisting issues that I am not refactoring as part of this PR.

### CODE REVIEW

File: spec/models/web_search_spec.rb [hunk 1]
```diff
@@ -67,149 +65,133 @@
...
+      expect(described_class.new(valid_options).cache_key).to eq("government (site:gov OR site:mil):{:query=>\"government\", :page=>5, :affiliate_id=>#{affiliate.id}}:BingV6")
     end
   end
 
   describe 'instrumenting search engine calls' do
     context 'when BingV6 is the engine' do
       before do
-        @valid_options = {query: 'government', affiliate: affiliate}
-        bing_search = BingV6WebSearch.new(@valid_options)
+        valid_options = { query: 'government', affiliate: affiliate }
```
Comment (jmax-fearless) [inline spec/models/web_search_spec.rb:86]: I don't think this is right; `valid_options` should be a context `let` here, not a local variable, yes?

**Review by jmax-fearless — CHANGES_REQUESTED** [review]
Only one comment, but I think it's a real issue; please address or convince me it's not.

File: spec/models/web_search_spec.rb (hunk 1 above)
Comment (MothOnMars) [reply spec/models/web_search_spec.rb:86]: Good catch - that was an oversight. Fixing...

**Review by jmax-fearless — APPROVED** [review]
LGTM

**Pull request merged**